packages/sortable-behavior-bundle/src/Behaviors/Sortable.php

```diff
@@ -22,6 +22,8 @@ trait Sortable
      * @Gedmo\SortablePosition
      * @ORM\Column(type="integer")
      */
+    #[Gedmo\SortablePosition]
+    #[ORM\Column('position', 'integer')]
```

Comment:
```suggestion
    #[ORM\Column(type: Types::INTEGER)]
```

And add a use for Types (is a dbal class)